Support erlang arm64 install

Currently get this error on a MacBook Pro M1:

```
 => ERROR [base 11/31] RUN install-tool erlang                                                                                                  23.3s
------
 > [base 11/31] RUN install-tool erlang:
#18 0.600 installing v2 tool erlang v24.3.4.1
#18 22.45 linking tool erlang v24.3.4.1
#18 23.02 qemu: uncaught target signal 11 (Segmentation fault) - core dumped
#18 23.03 /usr/local/bin/erl: line 6:   129 Segmentation fault      /opt/buildpack/tools/erlang/24.3.4.1/bin/erl "$@"
------
executor failed running [/bin/bash -c install-tool erlang]: exit code: 139
make: *** [project] Error 1
```

--- a/test/Dockerfile.arm64
+++ b/test/Dockerfile.arm64
@@ -26,7 +26,7 @@ RUN uname -p | tee | grep aarch64
 
 RUN touch /.dummy
 
-ARG IGNORED_TOOLS=erlang,elixir,php,composer
+ARG IGNORED_TOOLS=php,composer
 
 RUN prepare-tool all
 

